src/jobs/get-br-dependencies.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  exec:
+    type: executor
+  resource-class:
+    type: string
+    default: medium
+  hex-validate:
+    type: boolean
+    default: true
+  env-setup:
+    type: steps
+    default: []
+  save-dl-cache:
+    type: boolean
+    default: true
+executor: << parameters.exec >>
+resource_class: << parameters.resource-class >>
+steps:
+  - checkout
+  - steps: << parameters.env-setup >>
+  - install-elixir:
+      version: $ELIXIR_VERSION
+  - install-hex-rebar
+  - install-nerves-bootstrap
+  - mix-deps-get
+  - when:
+      condition: << parameters.hex-validate >>
+      steps:
+        - run:
+            name: Validate Hex package
+            command: mix hex.build
+  - restore-nerves-downloads
```

Comment:
Same here - I think we should restore early on

Reply:
Good call, I've updated this in 42c55e3.